src/agentscope/models/dashscope_model.py

```diff
@@ -398,6 +428,32 @@ def format(
             `List[dict]`:
                 The formatted messages.
         """
+        if self.api_type == "application":
```

Comment:
What's the requirement of the application api? Is it the same with the generation API?

Reply:
It requires the parameter `app_id` instead of `model`. Besides,  there are only minor differences, such as not supporting the `plugins` parameter, etc.